src/main/java/org/isf/opd/gui/OpdBrowser.java

```diff
@@ -379,7 +379,9 @@ private JButton getJDeleteButton() {
 				if (GeneralData.OPDEXTENDED) {
 					message = MessageBundle.formatMessage("angal.opd.deletefollowingopdextended.fmt.msg",
 							opd.getPatient().getName(),
-							opd.getCreatedDate().format(DATE_TIME_FORMATTER),
+							opd.getCreatedDate() == null
+									? opd.getDate().format(DATE_FORMATTER)
+									: opd.getCreatedDate().format(DATE_FORMATTER),
```

Comment:
Why to take into account the creation date OR the visit date? The latter should be always notnull and the former only for audit purpose